internal/api/grpc/user/v2/integration_test/user_test.go

```diff
@@ -2629,6 +2629,126 @@ func TestServer_ListAuthenticationMethodTypes(t *testing.T) {
 	}
 }
 
+func TestServer_ListAuthenticationFactors(t *testing.T) {
+	userIDWithoutAuth := Instance.CreateHumanUser(CTX).GetUserId()
+
+	userWithU2F := Instance.CreateHumanUser(CTX).GetUserId()
```

Comment:
The user creation could be move to the specific tests where you want to query the specific user, for parallel processing.